droid-command-line/src/main/java/uk/gov/nationalarchives/droid/command/action/CommandLineParam.java

```diff
@@ -257,6 +257,26 @@ public DroidCommand getCommand(CommandFactory commandFactory, CommandLine cli) {
         }
     },
 
+    /**
+     * Outputs the results as json.
+     */
+    JSON_OUTPUT("json", "json-output", I18N.JSON_OUTPUT) {
+        @Override
+        public DroidCommand getCommand(CommandFactory commandFactory, CommandLine cli) {
+            return null;
+        }
+    },
+
+    /**
+     * Outputs the results as csv.
+     */
+    CSV_OUTPUT("csv", "csv-output", I18N.JSON_OUTPUT) {
```

Comment:
this should be CSV_OUTPUT, I think in the I18N, the CSV output key is by mistake defined as XML